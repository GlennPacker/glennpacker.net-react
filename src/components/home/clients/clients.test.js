import React from 'react';
import { shallow } from 'enzyme';

import Clients from './index';

describe('clients', () => {
    let wrapper
    beforeAll(() => {
        wrapper = shallow(<Clients />);
    })

    it('should render', () => {
        expect(wrapper).not.toBeNull();
    })

    it('should render correctly compared to snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    })
})