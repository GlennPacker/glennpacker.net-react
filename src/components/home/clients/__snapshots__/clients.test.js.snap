// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clients should render correctly compared to snapshot 1`] = `
<div
  className="card"
>
  <Style
    rules={
      Object {
        ".client-card-body": Object {
          "minHeight": 450,
        },
        "ul": Object {
          "listStyle": "none",
          "paddingLeft": 0,
        },
      }
    }
    scopeSelector=""
  />
  <div
    className="card-header bg-info text-white"
  >
    Clients
  </div>
  <div
    className="card-body client-card-body"
  >
    <p
      className="card-text"
    >
      Previous clients in recent years:
    </p>
    <Connect(ClientContainer) />
  </div>
</div>
`;
